howso/analysis.amlg

```diff
@@ -182,6 +182,27 @@
 			(assign (assoc targeted_model "targetless"))
 		)
 
+		;if this is a time series Trainee, there are no targetless hyperparameters,
+		; and this analyze is targeted, emit a warning to the user that recommends
+		; a targeted analyze
+		(if (and
+				(!= (null) !tsTimeFeature)
+				(=
+					0
+					;the number of saved hyperparameter maps that are targetless
+					(size (filter (lambda (= ".targetless" (first (current_value))) ) !hyperparameterParamPaths))
+				)
+				(!= targeted_model "targetless")
+			)
+			(accum (assoc
+				analyze_warnings
+					(zip [(concat
```

Comment:
why not a single associate instead of a zip-list ?

Reply:
just was being silly I guess